src/models/test/StyleSheetManager.test.js

```diff
@@ -0,0 +1,76 @@
+// @flow
+/* eslint-disable react/no-multi-comp */
+import React from 'react'
+import { renderToString } from 'react-dom/server'
+import { shallow, mount } from 'enzyme'
+import styled, { ServerStyleSheet, StyleSheetManager } from '../../index'
+
+describe('StyleSheetManager', () => {
+  it('should render its child', () => {
+    const target = document.head
+    const Title = styled.h1`color: palevioletred;`
+    const child = <Title />
+    const renderedComp = shallow(
+      <StyleSheetManager target={target}>
+        { child }
+      </StyleSheetManager>
+    )
+    expect(renderedComp.contains(child)).toEqual(true)
+  })
+
+  // it('should use given stylesheet instance', (done) => {
```

Comment:
Why was this test commented out? It should work, right?

Reply:
Ah yes, I forgot to fix it.

It's now fixed, but just a note that I had to make it the first test in the file. I think it's because `StyleSheet.instance` is a singleton so it was using a `BrowserStyleSheet` if it was after one of the other tests? Not sure, but something to keep in mind.